blocks/models/huggingface/base.py

```diff
@@ -8,11 +8,15 @@
 from configs.constants import Const
 from datasets import ClassLabel, Dataset, Features, Value
 from sklearn.model_selection import train_test_split
-from transformers import (AutoModelForSequenceClassification, PreTrainedModel,
```

Comment:
So this happens because I have now configured vscode to organize imports (because it was suppose to also get rid of unused imports. 

Can you also set this?
In Vscode settings:
``    "editor.codeActionsOnSave": {
        "source.organizeImports": true
    },
``

Reply:
@itchingpixels

Reply:
can you put it in `.vscode/settings.json`? then it'll be done automatically